src/sections/DoYouNeedService/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import DoYouNeedWrapper from "./doYouNeedService.style";
+
+const DoYouNeedService = ({onClick}) => {
+    return (
+        <DoYouNeedWrapper>
+            <div id="content">
+                <div id="text">
+                    <span className="heading">Do You Need A <br/>Service Mesh?</span>
+                    <br/><br/>
+                    <span className="description">Let us work you through</span>
```

Comment:
```suggestion
                    <span className="description">Let's find out.</span>
```

Reply:
If it Ok to have same description and button text, I'd do that ✌️

Reply:
Fine ✌️